The code/Task2.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:io';
+
+class Task {
+  String descriptask;
+  bool isCompleted;
+
+  Task(this.descriptask, this.isCompleted);
+}
+
+class Todolist {
+  final List<Task> listTask = [];
+
+  void displayinfo() {
```

Comment:
use proper snake case
```suggestion
  void displayInfo() {
```